tgui/packages/tgui/interfaces/_FaxMachine.js

```diff
@@ -0,0 +1,286 @@
+              ) : (
+                "No stored paperwork to process."
+              )}
+            </Stack.Item>
+            <Stack.Item height={8}>
+              {selectedPaper ? (
+                <BlockQuote fontSize="14px">
+                  {selectedPaper.contents}
+                </BlockQuote>
+              ) : (
+                received_paperwork && received_paperwork.length > 0
+                && ("No paper selected.")
+              )}
+            </Stack.Item>
+            <Divider />
+            <Stack.Item height={3} mb={1}>
+              {!!selectedPaper && (
+                <Stack vertical align="center" >
+                  <Stack.Item>
+                    <b>
+                      To fulfill this paperwork, stamp accurately
+                      and answer the following:
+                    </b>
+                  </Stack.Item>
+                  <Stack.Item>
+                    <BlockQuote>
+                      {selectedPaper.required_answer}
+                    </BlockQuote>
+                  </Stack.Item>
+                </Stack>
+              )}
+            </Stack.Item>
+            <Stack.Item align="center" height={2}>
+              <Stack>
+                <Stack.Item>
+                  <Button
+                    color="good"
+                    disabled={!selectedPaper}
+                    content="Print Paper"
+                    tooltip="Print the selected paperwork. \
+                      This is how you stamp and process the paperwork."
+                    onClick={() => act('print_select_paperwork', {
+                      ref: selectedPaper.ref,
+                    })} />
+                </Stack.Item>
+                <Stack.Item>
+                  <Button
+                    disabled={!selectedPaper || emagged}
+                    content="Send Paper For Processing"
+                    tooltip="Send the selected paperwork \
+                      to your employer to check its \
+                      validity and receive your payment."
+                    onClick={() => act('check_paper', {
+                      ref: selectedPaper.ref,
+                    })} />
+                </Stack.Item>
+                <Stack.Item>
+                  <Button
+                    color="bad"
+                    disabled={!selectedPaper}
+                    content="Delete Paper"
+                    tooltip="Delete the selected paperwork from the machine."
+                    onClick={() => act('delete_select_paperwork', {
+                      ref: selectedPaper.ref,
+                    })} />
+                </Stack.Item>
+                <Stack.Item>
+                  <Button
+                    color={(!received_paperwork
+                      || received_paperwork.length <= 0) ? "good" : "caution"}
+                    disabled={!received_paperwork
+                      || received_paperwork.length <= 0}
+                    content="Print All Papers"
```

Comment:
The length of an array can't go below 0, so `===` works better here:
```suggestion
                    color={(!received_paperwork
                      || received_paperwork.length === 0) ? "good" : "caution"}
                    disabled={!received_paperwork
                      || received_paperwork.length === 0}
                    content="Print All Papers"
```